get.sh

```diff
@@ -625,41 +625,47 @@ fi
 
 checkRepoSHA()
 {
-	output_file="$TESTDIR/TKG/SHA.txt"
-	echo "$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $output_file"
-	$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $output_file
+	sha_file="$TESTDIR/TKG/SHA.txt"
+	testenv_file="$TESTDIR/testenv/testenv.properties"
+
+	echo "$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $sha_file"
+	$TESTDIR/TKG/scripts/getSHA.sh --repo_dir $1 --output_file $sha_file
+
+	echo "$TESTDIR/TKG/getTestenvProperties.sh --repo_dir $1 --output_file $testenv_file --repo_name $2"
+	$TESTDIR/TKG/getTestenvProperties.sh --repo_dir $1 --output_file $testenv_file --repo_name $2
```

Comment:
I do not think this path is correct. getTestenvProperties.sh is under `scripts` folder. https://github.com/adoptium/TKG/pull/248/files